recipes/flatbuffers/all/conanfile.py

```diff
@@ -36,20 +36,30 @@ def _has_flatc(self):
 
     def export_sources(self):
         copy(self, os.path.join("cmake", "FlatcTargets.cmake"), self.recipe_folder, self.export_sources_folder)
-        for p in self.conan_data.get("patches", {}).get(self.version, []):
-            copy(self, p["patch_file"], self.recipe_folder, self.export_sources_folder)
+        export_conandata_patches(self)
 
     def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
 
+    def _cmake_new_enough(self, required_version):
```

Comment:
There was conversations within the Conan team about conditioning the graph...

That not ideal but our desired solution doesn't exist  so maybe refrain from adding this check too much pkease... after the launch of 2.0 this month we can focus the team on these issues 🤞

Reply:
Mobile typing means terrible English 😑